package.json

```diff
@@ -1,66 +1,72 @@
-    "@changesets/cli": "^2.16.0",
-    "@nrwl/cli": "12.0.8",
-    "@nrwl/devkit": "12.0.8",
-    "@nrwl/eslint-plugin-nx": "12.0.8",
-    "@nrwl/jest": "12.0.8",
-    "@nrwl/linter": "12.0.8",
-    "@nrwl/node": "12.0.8",
-    "@nrwl/nx-plugin": "12.0.8",
-    "@nrwl/tao": "12.0.8",
-    "@nrwl/workspace": "12.0.8",
-    "@types/jest": "26.0.22",
-    "@types/node": "14.14.41",
-    "@typescript-eslint/eslint-plugin": "4.22.0",
-    "@typescript-eslint/parser": "4.22.0",
-    "babel-jest": "^26.6.3",
-    "dotenv": "8.2.0",
-    "eslint": "7.24.0",
-    "eslint-config-prettier": "8.2.0",
-    "jest": "26.6.3",
-    "prettier": "2.2.1",
-    "ts-jest": "26.5.5",
-    "ts-node": "~9.1.1",
-    "tslib": "^2.2.0",
-    "typescript": "4.2.4"
-  },
-  "dependencies": {
-    "@babel/core": "^7.13.16",
-    "@babel/plugin-syntax-optional-chaining": "^7.8.3",
-    "@babel/preset-env": "^7.13.15",
-    "@babel/preset-typescript": "^7.13.0",
-    "execa": "^5.0.0",
-    "regenerator-runtime": "^0.13.7"
-  }
+    "name": "nx-generators",
+    "version": "0.0.0",
+    "license": "MIT",
+    "scripts": {
+        "nx": "nx",
+        "start": "nx serve",
+        "build": "nx run-many --target=build --all",
+        "test": "nx run-many --target=test --all",
+        "lint": "nx workspace-lint && nx run-many --all --target=lint",
+        "e2e": "nx run-many --target=e2e --all",
+        "affected:apps": "nx affected:apps",
+        "affected:libs": "nx affected:libs",
+        "affected:build": "nx affected:build",
+        "affected:e2e": "nx affected:e2e",
+        "affected:test": "nx affected:test",
+        "affected:lint": "nx affected:lint",
+        "affected:dep-graph": "nx affected:dep-graph",
+        "affected": "nx affected",
+        "format": "nx format:write",
+        "format:write": "nx format:write",
+        "format:check": "nx format:check",
+        "update": "nx migrate latest",
+        "workspace-generator": "nx workspace-generator",
+        "dep-graph": "nx dep-graph",
+        "help": "nx help",
+        "preinstall": "npx only-allow pnpm",
+        "changesets": "changesets",
+        "generate:esbuild-options:build": "pnpx typescript-json-schema node_modules/esbuild/lib/main.d.ts BuildOptions -o libs/nx-esbuild/src/executors/build/schema.json && pnpx json -I -f libs/nx-esbuild/src/executors/build/schema.json -e 'this.cli=\"nx\"' && pnpx json -I -f libs/nx-esbuild/src/executors/build/schema.json -e 'this.title=\"ESBuild Build Executor\"'",
+        "generate:esbuild-options:serve": "pnpx typescript-json-schema node_modules/esbuild/lib/main.d.ts BuildOptions -o libs/nx-esbuild/src/executors/serve/schema.json && pnpx json -I -f libs/nx-esbuild/src/executors/serve/schema.json -e 'this.cli=\"nx\"' && pnpx json -I -f libs/nx-esbuild/src/executors/serve/schema.json -e 'this.title=\"ESBuild Serve Executor\"'",
+        "generate:esbuild-options": "pnpm run generate:esbuild-options:build && pnpm run generate:esbuild-options:serve"
```

Comment:
Added this, it generates the JSON schema NX uses from the ESBuild typescript interfaces. Make it super easy for us to update